plugins/modules/container_distribution.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# copyright (c) 2021, Mark Goddard
+# copyright (c) 2020, Jacob Floyd
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+---
+module: container_distribution
+short_description: Manage container distributions of a pulp api server instance
```

Comment:
"of a pulp api server instance"
I never liked this piece. If you (as a fresh contributor) have a better idea to formulate this, bring it on.

Reply:
how about "in a Pulp service"?

Reply:
I can do that in a separate change for all the modules, once this one lands.